src/containers/Tenant/Query/QuerySettingsDialog/QuerySettingsDialog.scss

```diff
@@ -0,0 +1,72 @@
+.ydb-query-settings-dialog {
+    &__dialog-row {
+        display: flex;
+        align-items: flex-start;
+
+        & + & {
+            margin-top: var(--g-text-body-1-line-height);
+        }
+    }
+    &__field-title {
+        flex: 4;
+
+        margin-right: 12px;
+
+        font-weight: 500;
+        line-height: 28px;
+        white-space: nowrap;
+
+        &.required {
+            &::after {
+                content: '*';
+
+                color: var(--g-color-text-danger);
+            }
+        }
+    }
+
+    .g-dialog-footer__bts-wrapper {
+        width: 100%;
+    }
+
+    &__dialog-body {
+        padding-top: 24px;
+    }
+
+    &__control-wrapper {
+        display: flex;
+        flex: 6;
+    }
+
+    &__timeout {
+        width: 33.3%;
+        margin-right: 8px;
+    }
+
+    &__timeout-suffix {
+        display: flex;
+        align-items: center;
+
+        color: var(--g-color-text-secondary);
+    }
+
+    &__documentation-link {
```

Comment:
Should we get rid of it?

Reply:
why?

Reply:
Is was just for not to forget to clarify about docs link :)